BoldGrid/crio
# Cannot work with widgets in customizer on Primary Sidebar

ISSUE: Resolves #91 

Cannot work with widgets in customizer on Primary Sidebar

## Test Files ##

{LINK TO TEST ZIP FILES HERE}

## NOTES TO TESTER ##
### Please leave comments regarding issues found in testing directly on the issue linked above, not the Pull Request. This helps ease the process of reviewing the testing of multiple PRs in a given project from the project view. ###
### Please remember to move this item from 'Needs Review' to either 'In Progress' or 'Awaiting Release' depending on the results of your testing ###

## Testing Instructions ##

1. Install the test zip files linked above.
2. Enable a sidebar on a page ( such as a blog ).
3. Open customizer, navigate to the page with the sidebar and add a widget to the 'Primary Sidebar'
4. Ensure that after adding a widget, that the 'Primary Sidebar' in the 'Widgets' panel of the customizer is not removed from the  panel.
